support-frontend/assets/pages/paper-subscription-checkout/components/thankYou.jsx

```diff
@@ -32,53 +33,68 @@ import { type FormFields, getFormFields } from 'helpers/subscriptionsForms/formF
 type PropTypes = {
     ...FormFields,
     isPending: boolean,
+    useDigitalVoucher: boolean,
 };
 
+// ----- Map State/Props ----- //
+
+function mapStateToProps(state: WithDeliveryCheckoutState) {
+  return {
+    ...getFormFields(state),
+    useDigitalVoucher: state.common.settings.useDigitalVoucher,
```

Comment:
presumably this is existing, but I didn't think useDigitalVoucher was state, more a universal constant only changed with a deploy

Reply:
It is a constant, but it's incorporated into the Redux state despite not changing, I think just to make accessing it more convenient.